docs/CONTRIBUTING.md

```diff
@@ -208,6 +209,12 @@ with the number of lines changed.  Therefore if your pull request is
 too large, we may ask you to break it up into several smaller pull
 requests.
 
+If you would like feedback on a pull request prior to it being ready
+for formal review, please open it
+[in draft mode](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
+and [request reviews][request-reviews] from whomever you wish to get feedback
+from. As long as the PR is in draft mode, no one should review the pull request.
```

Comment:
"As long as the PR is in draft mode, it will not be reviewed, aside from the feedback requested."

Otherwise, this sounds like you're saying no one should look at it.